ext/GDBM_File/typemap

```diff
@@ -11,6 +11,7 @@ SDBM_File		T_PTROBJ
 ODBM_File		T_PTROBJ
 DB_File			T_PTROBJ
 DBZ_File		T_PTROBJ
+gdbm_count_t            T_COUNT
```

Comment:
Is there any specific reason for not using `T_UV` here?

Reply:
The initial reason was that gdbm_count_t is not a fixed typedef, but is determined at configure time. But I'm going to change it to a fixed type in future versions of gdbm anyway, so yes, it's better to use T_UV  here.